web/src/admin/providers/saml/SAMLProviderViewPage.ts

```diff
@@ -257,13 +257,7 @@ export class SAMLProviderViewPage extends AKElement {
         if (!this.provider) {
             return html``;
         }
-        return html`${
```

Comment:
I do not know why sometimes Prettier makes the extra-long ones and doesn't fix it.